examples/elasticsearch/es-rc.yaml

```diff
@@ -48,5 +49,3 @@ spec:
           name: storage
       volumes:
       - name: storage
-        source:
-          emptyDir: {}
```

Comment:
No sure where you getting this from but it should be:
```
      volumes:
          - emptyDir:
              medium: ""
            name: "storage"
```
whenever a pod spec descriptor is declared, including documentation.